src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +296,37 @@
     }
   }
 
+  .pf-c-button {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+    height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
```

Comment:
I saw some designs (that may or may not make their way into PF) that would imply we need to support this button focus state potentially being pill shaped if there is a lot of content in the button. Here are a couple of screenshots.

I wonder if we could support that without much effort in the original implementation? If not it's always an enhancement we could make later if needed.

<img width="89" alt="Screen Shot 2020-01-10 at 12 16 39 PM" src="https://user-images.githubusercontent.com/35148959/72176132-65461900-33a3-11ea-8027-40fd03db093b.png">

Reply:
Assuming the badge and notification would be part of the same button, we could possibly have a selector for multiple elements in the button and then change the width/height/border-radius accordingly. Maybe there would be a variation of the notification badge that supports the badge component too and handles the spacing there. Either way, I think we can just update properties based on whats inside of the notification badge, with the current implementation.

Reply:
That's one way to do it, though I wonder if it could just adapt automatically. I changed the button height/width to min-height/min-width and gave the button sm (8px) padding here. Not sure if something like that would work?

<img width="159" alt="Screen Shot 2020-01-10 at 4 27 09 PM" src="https://user-images.githubusercontent.com/35148959/72190890-52910b80-33c6-11ea-870e-4af381117ad7.png">
<img width="146" alt="Screen Shot 2020-01-10 at 4 27 03 PM" src="https://user-images.githubusercontent.com/35148959/72190892-52910b80-33c6-11ea-9d47-54cc79bc897e.png">
<img width="146" alt="Screen Shot 2020-01-10 at 4 26 56 PM" src="https://user-images.githubusercontent.com/35148959/72190893-52910b80-33c6-11ea-95e3-c608ae06454a.png">

Reply:
oh thats a good point, seems easier.